app/store/ocean.ts

```diff
@@ -29,25 +29,16 @@ export const ocean = createSlice({
     queueTransaction: (state, action: PayloadAction<OceanTransaction>) => {
       state.transactions = [...state.transactions, action.payload]
     },
-    closeOceanInterface: (state) => {
-      state.transactions = []
-      state.err = undefined
-    },
     setError: (state, action: PayloadAction<Error>) => {
       state.err = action.payload
     },
     popTransaction: (state) => {
-      if (state.transactions.length > 1) {
-        state.transactions.shift()
-        state.transactions = [...state.transactions]
-      }
+      state.transactions.shift()
```

Comment:
This solely serves as job queue. Remove them all even is last one. UI kept the last in state to display until manual dismissed.